grunt-configs/concat.js

```diff
@@ -9,6 +9,14 @@ module.exports = function (grunt, options) {
                 options.staticTargetDir + 'javascripts/boot.js'
             ],
             dest: options.staticTargetDir + 'javascripts/app.js'
+        },
+
+        ie8: {
+            src: [
+                options.staticPublicDir + 'javascripts/components/es5-shim/es5-shim.js',
+                options.staticPublicDir + 'javascripts/components/html5shiv/html5shiv.js'
+            ],
+            dest: options.staticPublicDir + 'javascripts/ie8polyfills.js'
```

Comment:
sorry to be a pain, but how about calling this `es-html-5-shiv` or something like that? not a blocker, but I wonder if this would be more explicit to any interested readers?

```html
<!--[if lt IE 9]>
<script src="@Static("javascripts/es5-html5-shivs.js")"></script>
```